src/shared_modules/utils/flatbuffers/CMakeLists.txt

```diff
@@ -7,6 +7,10 @@ enable_testing()
 set(CMAKE_CXX_STANDARD 17)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 
+if (NOT SRC_FOLDER)
+    get_filename_component(SRC_FOLDER     ${CMAKE_SOURCE_DIR}/../../../ ABSOLUTE)
```

Comment:
❓ Can we avoid relatives' paths?

Reply:
I've done the same here

https://github.com/wazuh/wazuh/pull/28345/files

The thing is that the `SRC_FOLDER` variable is defined at the root CMakeLists
https://github.com/wazuh/wazuh/blob/enhancement/27897-add-inventory-workflows-automation/src/CMakeLists.txt#L18

When it isn't, then it means that only this module is being built.
In that case, we either define the variable in a workflow or we get the path to `src` this way.

Reply:
Excellent, solved :)